content/blog/2021-04-19-experiment-refs.md

```diff
@@ -0,0 +1,277 @@
+---
+title: Utilizing Custom Git References in DVC
```

Comment:
Should we include ML experiment in the title to make it more general?
Like `Git Custom References for ML experiments`

Reply:
I'm fine with changing the title but we'd need a new cover image @jendefig

Reply:
This one looks good to me. It should be 100% match. I'm fine with the original one if you think DVC is important.

An idea if you prefer keep DVC - `DVC: Git Custom References for ML experiments`